.mailmap

```diff
@@ -0,0 +1,10 @@
+Michele Peresano <michele.peresano@cea.fr> Michele Peresano <michele.peresano@.cea.fr>
+Michele Peresano <michele.peresano@cea.fr> Michele Peresano <peresano.michele@gmail.com>
+Julien Lefaucheur <julien.lefaucheur@obspm.fr>
+Thierry Stolarczyk <thierry.stolarczyk@cea.fr>
+Gaia Verna <gverna94@gmail.com> gaia-verna <51763341+gaia-verna@users.noreply.github.com>
+Gaia Verna <gverna94@gmail.com> gaia-verna <gverna94@gmail.com>
+David Landriu <landriu@sapcta.extra.cea.fr>
+Karl Kosack <kosack@users.noreply.github.com>
```

Comment:
could change this to:
```
Karl Kosack <karl.kosack@cea.fr> Karl Kosack <kosack@users.noreply.github.com>
```
as in ctapipe. Hopefully we don't get too much spam from having real emails here, but it's been like that in ctapipe for a while